umbral-pre-wasm/example-bundler/yarn.lock

```diff
@@ -0,0 +1,3819 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can you add some instructions on how it was generated, or perhaps, add a `lock` command in `Makefile`?

Reply:
This is a dependency resolution file generated by `yarn`, similar to how `npm` generates `package.lock`. It can be managed indirectly by `yarn` commands, such as `yarn add`, `yarn remove` etc. IMHO it doesn't need additional instructions or `Makefile` commands as it is a part of the well-defined and well-known workflow.